### PR TITLE
AT_04.23.01 | Create booking page > Verify "Passenger details" label is present and visible.

### PR DESCRIPTION
**Pull request by Janyl17**
https://trello.com/c/JtvnI5T0/268-at042301-create-booking-page-verify-passenger-details-label-is-present-and-visible

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/// <reference types="Cypress" />
+
+import CreateBookingPage from "../../../../pageObjects/CreateBookingPage"
+
+const createBookingPage = new CreateBookingPage();
+
+describe('US_04.23 | Create booking page > Passengers details default UI', () => {
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js:7]: Could you change the describe, please => 'US_04.23 | Passengers details default UI'

File: cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/// <reference types="Cypress" />
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js:1]: Could you change the file's name, please => us_04.23_PassengersDetailsDefaultUI.cy.js

File: cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+const createBookingPage = new CreateBookingPage();
+
+describe('US_04.23 | Create booking page > Passengers details default UI', () => {
+
+    const AGENT = Cypress.env('agent');
+
+    beforeEach(function () {
+        cy.visit('/');
+
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js:13]: Please remove empty line

File: cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    });
+    
+    it('AT_04.23.01 | Verify Passenger details label is visible', () => {
+        cy.login(AGENT.email, AGENT.password);
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js:17]: Please move to describe

File: cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js (hunk 1 above)
Comment (Janyl17) [reply cypress/e2e/testUiAndFunction/createBookingPage/passengersDetailsSection/us_04.23_CreateBookingPage _PassengersDetailsDefaultUI.cy.js:7]: Thank you, pls review it again.

**Review by Rom-T — APPROVED** [review]
LGTM

**Pull request merged**